deployment/terraform/coordinator.go

```diff
@@ -0,0 +1,125 @@
+package terraform
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"github.com/mattermost/mattermost-load-test-ng/coordinator"
+	"github.com/mattermost/mattermost-load-test-ng/coordinator/agent"
+	"github.com/mattermost/mattermost-load-test-ng/coordinator/cluster"
+	"github.com/mattermost/mattermost-load-test-ng/coordinator/performance"
+	"github.com/mattermost/mattermost-load-test-ng/coordinator/performance/prometheus"
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/ssh"
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+// Start starts the coordinator that is deployed.
+func (t *Terraform) StartCoordinator() error {
+	if err := t.preFlightCheck(); err != nil {
+		return err
+	}
+
+	output, err := t.getOutput()
+	if err != nil {
+		return err
+	}
+
+	if len(output.Agents.Value) == 0 {
+		return fmt.Errorf("there are no agent instances to run the coordinator")
+	}
+	ip := output.Agents.Value[0].PublicIP
+
+	loadtestConfig := t.generateLoadtestAgentConfig(output)
+
+	var loadAgentConfigs []agent.LoadAgentConfig
+	for _, val := range output.Agents.Value {
+		loadAgentConfigs = append(loadAgentConfigs, agent.LoadAgentConfig{
+			Id:             val.Tags.Name,
+			ApiURL:         "http://" + val.PrivateIP + ":4000",
+			LoadTestConfig: loadtestConfig,
+		})
+	}
+
+	extAgent, err := ssh.NewAgent()
+	if err != nil {
+		return err
+	}
+	sshc, err := extAgent.NewClient(ip)
+	if err != nil {
+		return err
+	}
+
+	mlog.Info("Setting up coordinator", mlog.String("ip", ip))
+	clusterConfig := coordinator.Config{
```

Comment:
Wondering if here we can leverage `coordinator.GetConfig()` that's coming with https://github.com/mattermost/mattermost-load-test-ng/pull/198.  
This is to avoid the hard-coded duplication of unrelated settings.

Reply:
So, you basically suggest to use local config right? Even so, we need to update it for the ip addresses etc, but it's good to have. A separate PR maybe?

Reply:
Yes, the idea is just replace what's needed for the deployment to work. Fine with making it a separate PR.